src/Utilities/functions.jl

```diff
@@ -778,6 +778,11 @@ function operate(op::typeof(+), ::Type{T}, f, g, h, args...) where T
     return operate!(+, T, operate(+, T, f, g), h, args...)
 end
 
+# Unary +
+function operate(op::typeof(+), ::Type{T}, f::MOI.AbstractFunction) where T
+    return f
```

Comment:
The docstring for `operate` doesn't say what `T` is used for. Why is it ignored most of the time?

Reply:
It is the coefficient type. It is useful when summing two SingleVariable for instance